wrangler-core/src/main/java/io/cdap/directives/column/SetType.java

```diff
@@ -258,9 +258,8 @@ private Object convertType(String toType, Object object) throws Exception {
 
       default:
         throw new DirectiveExecutionException(
-          String.format("Unknown data type '%s' found in the directive. " +
-                  "Accepted types are: int, short, long, double, boolean, string, bytes", toType)
-        );
+          NAME, String.format("Unknown data type '%s' found in the directive. " +
```

Comment:
Can we enhance this message to include the column name? Something like

Column '%s' is of unsupported type '%s'. Supported types are: ...